docs/using.rst

```diff
@@ -507,8 +507,9 @@ The following files are available:
   <http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_certificate_key>`_.
 
 ``fullchain.pem``
-  All certificates, **including** server certificate. This is
-  concatenation of ``cert.pem`` and ``chain.pem``.
```

Comment:
Yeah, this line was confusing with the reordering, and also unnecessary. If you feel like it's important information, I could add it below in the `cert.pem` / `chain.pem` section, something like "If you concatenate cert.pem and chain.pem, you would identical contents as fullchain.pem." But I'm not sure that's useful to anyone.